### PR TITLE
project: note on project roles

### PR DESCRIPTION
**Pull request by shwstppr**
Addresses https://github.com/apache/cloudstack/issues/9071

----
📚 Documentation preview 📚: https://cloudstack-documentation--403.org.readthedocs.build/en/403/

### CODE REVIEW

**Review by Pearl1594 — APPROVED** [review]
LGTM

File: source/adminguide/projects.rst [hunk 1]
```diff
@@ -288,6 +288,16 @@ to add members in CloudStack, but only one way is enabled at a time:
...
 
 
+Working with Project Roles
+--------------------------
+CloudStack allows adding project members with a desired project role. A
+project role will be assigned to the member in addition to their base
+account role. Project Roles are retrictive in nature and can be used to
+further restrict certain API access to the members within the project.
+Project roles can be created or managed using `Project roles` tab in the
+project details UI.
```
Comment (vishesh92) [inline source/adminguide/projects.rst:298]: IMO, this seems a little confusing. It will be good to add an example to explain this.

Comment (shwstppr) [reply source/adminguide/projects.rst:298]: I don't have any example. IMO, I don't see any utility of project roles in its current form. So, I just added what was informed to me. Happy to accept changes or close the PR for better changes.

Comment (vishesh92) [inline source/adminguide/projects.rst:298]:
```suggestion
further restrict certain API access to the members within the project.
It is important to note that a project role cannot be used to elevate an
existing user's permissions. Project roles can be created or managed
using `Project roles` tab in the project details UI.
```

Comment (vishesh92) [reply source/adminguide/projects.rst:298]: @shwstppr Just added the below line to be more clear.
```
 It is important to note that a project role cannot be used to elevate an existing user's permissions.
```

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Review by vishesh92 — APPROVED** [review]
lgtm

**Pull request merged**